libs/crypto/keys_manager_impl.cpp

```diff
@@ -73,55 +64,37 @@ namespace iroha {
     return true;
   }
 
-  bool KeysManagerImpl::loadFile(const std::string &filename,
-                                 std::string &res) {
-    std::ifstream file(filename);
+  boost::optional<std::string> KeysManagerImpl::loadFile(
+      const boost::filesystem::path &path) const {
+    auto file_path = path.string();
+    std::ifstream file(file_path);
     if (not file) {
-      log_->error("Cannot read '" + filename + "'");
-      return false;
+      log_->error("Cannot read '" + file_path + "'");
+      return {};
     }
-    file >> res;
-    return true;
-  }
-
-  boost::optional<Keypair> KeysManagerImpl::loadKeys() {
-    std::string pub_key;
-    std::string priv_key;
-
-    if (not loadFile(account_name_ + kPublicKeyExtension, pub_key)
-        or not loadFile(account_name_ + kPrivateKeyExtension, priv_key))
-      return boost::none;
-
-    Keypair keypair = Keypair(PublicKey(Blob::fromHexString(pub_key)),
-                              PrivateKey(Blob::fromHexString(priv_key)));
 
-    return this->validate(keypair) ? boost::make_optional(keypair)
-                                   : boost::none;
+    std::string contents;
+    file >> contents;
+    return boost::make_optional<std::string>(contents);
```

Comment:
`return contents;` should be OK, there should be an implicit ctor for the optional.